test/scraping/cycling/procyclingstats/raceStages.test.js

```
@@ -1,34 +1,76 @@
+    stagesData = await Bun.file(data.stages.json).json();
+    const raceStages = new RaceStages();
+    await raceStages.update(stagesData);
+  });
 
-    html = await input.text();
-    expectedResults = await output.json();
+  afterAll(async () => {
+    process.env.DATA_DIR = originalDataDir;
+    try {
+      await Bun.file(`${TEST_DATA_DIR}/raceStages.csv`).delete();
+    } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)